.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        scala: [2.12.19]
+        scala: [2.12.19, 3.3.3]
```

Comment:
Given that this repo already have a lot of jobs, I think we should build both sbt 1.x and 2.x in one job.

Reply:
done